packages/react-components/src/templates/SharedResearchOutput.tsx

```diff
@@ -162,7 +171,33 @@ const SharedResearchOutput: React.FC<SharedResearchOutputProps> = ({
             }`}
             onSave={() => toggleReview(!reviewRequestedBy)}
             onCancel={() => {
-              setDisplayModal(false);
+              setDisplayReviewModal(false);
+            }}
+          />
+        )}
+        {displayPublishModal && (
+          <ConfirmModal
+            title={'Publish output for the whole hub?'}
+            description={
+              <>
+                {`All ${
+                  association === 'working group' ? 'working group' : 'team'
+                } members listed on this output will be notified and all
+                CRN members will be able to access it. If you want to switch to
+                draft after the output was published you need to contact`}
+                <Link href={mailToSupport()}> techsupport@asap.science</Link>.
```

Comment:
I don't like that we have an email address that could change here. Can we not export it from somewhere?

Reply:
Extracted the mail into a const, as it was also hardcoded in mailToSupport too, so now we only have to change it in 1 place if it does change in the future